docs/website/docs/guides/deployment-configurations/cpu.md

```diff
@@ -109,11 +109,13 @@ With the requirements out of the way, we can now compile a model and run it.
 
 --8<-- "docs/website/docs/guides/deployment-configurations/snippets/_iree-import-onnx-mobilenet.md"
 
-Then run the following command to compile with the `llvm-cpu` target:
+Then run the following command to compile with the `local` device and `llvm-cpu`
+target compilation backend:
 
 ``` shell hl_lines="2-3"
```

Comment:
A new line was added here, so highlight lines 2-4 instead of just lines 2-3
```suggestion
``` shell hl_lines="2-4"
```